bin/pre-commit

```diff
@@ -11,4 +11,4 @@ fi
 
 find . \( -name '*.php' \) -exec php -lf {} \;
 
-phpunit --filter WordPress "$phpcs_root/tests/AllTests.php"
+PHPCS_DIR=$phpcs_root phpunit --bootstrap="./Test/phpcs3-bootstrap.php" --filter WordPress "$phpcs_root/tests/AllTests.php"
```

Comment:
The WPCS repo root has already been traversed into via `cd "$(git rev-parse --show-toplevel)"`, so no need for `../Test`.

Without the `PHPCS_DIR` environment var, I get:

```
Uh oh... can't find PHPCS. Are you sure you are using PHPCS 3.x ?

If you use Composer, please run `composer install`.
Otherwise, make sure you set a `PHPCS_DIR` environment variable in your phpunit.xml file
pointing to the PHPCS directory.

Please read the contributors guidelines for more information:
https://is.gd/WPCScontributing
```